examples/scalajs-react/src/main/scala/todomvc/CTodoList.scala

```diff
@@ -0,0 +1,148 @@
+package todomvc
+
+import japgolly.scalajs.react.ScalazReact._
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.extra._
+import japgolly.scalajs.react.extra.router2.RouterCtl
+import japgolly.scalajs.react.vdom.prefix_<^._
+import org.scalajs.dom.ext.KeyCode
+
+import scala.scalajs.js
+import scalaz.effect.IO
+import scalaz.std.anyVal.unitInstance
+import scalaz.syntax.semigroup._
+
+object CTodoList {
+
+  case class Props private[CTodoList] (ctl: RouterCtl[TodoFilter], model: TodoModel, currentFilter: TodoFilter)
+
+  case class State(todos: Seq[Todo], editing: Option[TodoId])
+
+  /**
+   * These specify when it makes sense to skip updating this component (see comment on `Listenable` below)
+   */
+  implicit val r1 = Reusability.fn[Props]((p1, p2) ⇒ p1.currentFilter == p2.currentFilter)
```

Comment:
The unicode arrows are fun, but inconsistent with the rest of the app. Could we stick with one please?